test/torchaudio_unittest/functional/autograd_impl.py

```diff
@@ -250,6 +252,59 @@ def test_bandreject_biquad(self, central_freq, Q):
         Q = torch.tensor(Q)
         self.assert_grad(F.bandreject_biquad, (x, sr, central_freq, Q))
 
+    @parameterized.expand(
+        [
+            (True,),
+            (False,),
+        ]
+    )
+    def test_compute_power_spectral_density_matrix(self, use_mask):
+        torch.random.manual_seed(2434)
+        specgram = torch.rand(4, 10, 5, dtype=torch.cfloat)
+        if use_mask:
+            mask = torch.rand(10, 5)
+        else:
+            mask = None
+        self.assert_grad(F.compute_power_spectral_density_matrix, (specgram, mask))
+
+    def test_compute_mvdr_weights_souden(self):
+        torch.random.manual_seed(2434)
+        channel = 4
+        psd_speech = torch.rand(64, channel, channel, dtype=torch.cfloat)
+        psd_noise = torch.rand(64, channel, channel, dtype=torch.cfloat)
```

Comment:
Autograd tests are one of the most time-consuming ones. I think batch size 3 is suffice.

Reply:
or is this frame length?

Reply:
64 is the frequency dimension. We can reduce it to 5 or 10.